### ISSUE TITLE
Blacklist Climbable blocks

### ISSUE THREAD

**Issue by NewbulaCo** [genandnic/Wall-Jump#47]
Suggestion: 
Having a config option to blacklist certain block itemIDs would be very useful. I am building obstacle courses that would make great use of this config. Thank you.

**Comment by LLjo** [comment]
I agree. I thought it was a future at one point but maybe another mod implemented that.
